build.gradle

```diff
@@ -19,7 +19,7 @@ task wrapper(type: Wrapper) {
 }
 
 buildscript {
-  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/indigo/buildscript.gradle"
+  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/kinetic/buildscript.gradle"
```

Comment:
This is probably something that could almost be made constant and ease the release process. Damon was looking into it at some point. However probably not high priority with this (last) release. Couldn't do immediately as it points to shifting changes in the rosjava bootstrap libraries.

@adamantivm could you:

* Could you make an issue about this in rosjava bootstrap?
* Check that the kinetic version of the buildscript points to the right rosjava bootstrap version as a dependency